common/web/types/build.sh

```diff
@@ -82,9 +82,26 @@ function do_configure() {
 function do_test() {
   eslint .
   tsc --build test
-  readonly C8_THRESHOLD=50
-  c8 -skip-full --reporter=lcov --reporter=text --lines $C8_THRESHOLD --statements $C8_THRESHOLD --branches $C8_THRESHOLD --functions $C8_THRESHOLD mocha "${builder_extra_params[@]}"
-  builder_echo warning "Coverage thresholds are currently $C8_THRESHOLD%, which is lower than ideal."
+
+  readonly C8_THRESHOLD=60
+  # Exclude files from coverage analysis that only define types:
+  exclude=(\
+    src/keyboard-object.ts \
+    src/lexical-model-types.ts \
+    src/outputTarget.interface.ts \
+    src/*.d.ts \
+    src/main.ts \
+    src/schemas/* \
+    src/schema-validators.ts \
+    src/schemas.ts \
+  )
+  # shellcheck disable=SC2068
+  c8 --skip-full --reporter=lcov --reporter=text --lines "${C8_THRESHOLD}" \
+    --statements "${C8_THRESHOLD}" --branches "${C8_THRESHOLD}" \
+    --functions "${C8_THRESHOLD}" ${exclude[@]/#/--exclude } \
+    mocha "${builder_extra_params[@]}"
```

Comment:
It would be good to put those exclusions into a .c8rc.json file instead of burdening the build.sh. I'd leave the coverage flags in build.sh because we use that for reporting the warning as well.